ui/src/app/lib/atlasmap-data-mapper/common/config.types.ts

```diff
@@ -15,7 +15,7 @@
 */
 
 export enum DocumentType {
-  JAVA = 'Java',
+  JAVA = 'JAVA',
   JAVA_ARCHIVE = 'JAR',
```

Comment:
is it possible JAVA_ARCHIVE to be used for uri, causes `atlas:jar'?

Reply:
Is  atlas:jar:id valid?

Reply:
nope, it should be `atlas:java:id` as it's still supposed to be handled with JavaModule.

Reply:
I'll push the fix for the first comment - if atlas.jar is required we can catch that in 7.4

Reply:
ah no we don't need it, I just worry if in any case DocumentType.JAVA_ARCHIVE is passed into ConfigModel, and then it causes "atlas:jar:id" uri which is wrong.